app-tasks/usr/local/airflow/airflow.cfg

```diff
@@ -1,299 +0,0 @@
-[core]
-# The home folder for airflow, default is ~/airflow
-airflow_home = /usr/local/airflow
-
-# The folder where your airflow pipelines live, most likely a
-# subfolder in a code repository
-# This path must be absolute
-dags_folder = /opt/raster-foundry/app-tasks/dags
-
-# The folder where airflow should store its log files
-# This path must be absolute
-base_log_folder = /usr/local/airflow/logs
```

Comment:
Some references to this in the `.gitignore` file.

Reply:
Resolved by https://github.com/raster-foundry/raster-foundry/pull/2558/commits/238d6249fe48e60dcbb4bcc2b09103dd01e0b069